### PR TITLE
T3T1 UI: share words: simple slide animation

### LINKED ISSUE #3778: Animations between words during wallet creation

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #3778.

### CODE REVIEW

File: core/embed/rust/src/ui/util.rs [hunk 1]
```diff
@@ -179,6 +180,32 @@ macro_rules! include_icon {
     };
 }
 
+pub const SLIDE_DURATION: Duration = Duration::from_millis(333);
```
Comment (obrusvit) [inline core/embed/rust/src/ui/util.rs:183]: Let's be explicit with the unit `SLIDE_DURATION_MS`

File: core/embed/rust/src/ui/model_mercury/component/share_words.rs [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
         model_mercury::component::{Footer, SwipeDirection},
         shape,
         shape::Renderer,
+        util,
     },
 };
 use heapless::{String, Vec};
 
 const MAX_WORDS: usize = 33; // super-shamir has 33 words, all other have less
+const ANIMATION_DURATION: Duration = Duration::from_millis(166);
```
Comment (obrusvit) [inline core/embed/rust/src/ui/model_mercury/component/share_words.rs:20]: Same here, pls `ANIMATION_DURATION_MS`

File: core/embed/rust/src/ui/lerp.rs [hunk 3]
```diff
@@ -71,6 +71,8 @@ impl_lerp_for_uint!(u8);
 impl_lerp_for_uint!(u16);
 impl_lerp_for_uint!(u32);
 
+impl_lerp_for_int!(f32);
```
Comment (obrusvit) [inline core/embed/rust/src/ui/lerp.rs:74]: `for_int!`?

File: core/embed/rust/src/ui/util.rs (hunk 1 above)
Comment (mmilata) [reply core/embed/rust/src/ui/util.rs]: well the unit is in the constructor but :shrug: 87a76234de2eb419c85ce1ba025b208e6e9569ad

File: core/embed/rust/src/ui/model_mercury/component/share_words.rs (hunk 2 above)
Comment (mmilata) [reply core/embed/rust/src/ui/model_mercury/component/share_words.rs]: 87a76234de2eb419c85ce1ba025b208e6e9569ad

File: core/embed/rust/src/ui/lerp.rs (hunk 3 above)
Comment (mmilata) [reply core/embed/rust/src/ui/lerp.rs]: 87a76234de2eb419c85ce1ba025b208e6e9569ad

**Comment by mmilata** [comment]
build should be fixed now, really looking forward to a passing CI baseline

**Review by obrusvit — APPROVED** [review]

**Pull request merged**